lib/LedgerSMB/Sysconfig.pm

```diff
@@ -163,7 +167,7 @@ def 'force_username_case',
 
 def 'max_post_size',
     section => 'main',
-    default => 1024 * 1024,
+    default => MEGABYTE,
```

Comment:
I suggest not  doing a no-critic; you are right, but the person who does not see that is not wrong.

Although the codebase has its odd quirks and warts at present it has four  no-critic directives (soon to be three).
I have suggested a fairly loose application of no-critic on Lsmb.  I was wrong; we should avoid `## no critic` like the plague.